source/_components/cast.markdown

```diff
@@ -30,9 +30,54 @@ For example, Cast devices can only be discovered if they are on the same subnet
 cast:
   media_player:
   - host: 192.168.1.10
+  - port: 8009
 ```
 
 Configuration variables:
 
 - **host** (*Optional*): Use only if you don't want to scan for devices.
+- **port** (*Optional*): Specify chromecast port number - useful to define Chromecast Audio groups
 - **ignore_cec** (*Optional*) A list of Chromecasts that should ignore CEC data for determining the active input. [See the upstream documentation for more information.](https://github.com/balloob/pychromecast#ignoring-cec-data)
+
+### {% linkable_title Chromecast Audio Groups and static configuration %}
+Chromecasts and its Audio Groups are added automatically as long as [the discovery component]({{site_root}}/components/discovery/) is used to discover your Chromecast devices, Home Assistant is running in the same network as your Chromecast devices, and you don't have any Chromecast devices configured manually in your *configuration.yaml* file (the configuration includes only `cast:` and no static devices).
+
+If for you wish to define your devices manually (i.e.: when running Home Assistant in another network), these groups can be manually added by specifying the group master's IP and the group's port.
+
+**Caution:** The group is always "tracked" by a single master device. The master of a group could potentially change, after which a manual update of the configuration is required.
+
+#### {% linkable_title Discovering Chromecasts and Audio Groups %}
+The information to use in a static config could be found by e.g. by running a tool called **netdisco** manually in your local network.
+
+##### {% linkable_title Native installation %}
+On a system in your network with python and pip installed, the following commands can be run:
+```
+pip install netdisco
+python -m netdisco
+```
+and look for the devices listed under `google_cast:`.
+
+##### {% linkable_title Virtualenv installation %}
+If your Home Assistant installation is running in a [virtualenv](https://www.home-assistant.io/docs/installation/virtualenv/), you'll need to activate the virtualenv before running the above commands, e.g.:
```

Comment:
One problem here, sorry, is that it doesn't mention running it as the user you run Home Assistant as. Anybody following the RPi guide, or Hassbian, or using anything other than their default user will break their install here.

Reply:
@DubhAd 
Good catch. I'm not that familiar with anything non-hassio tbh :-)

Can you recheck please?
Also, any suggestions on what to advice hass.io users...?